SimFastTiming/FastTimingCommon/src/BTLDeviceSim.cc

```diff
@@ -82,11 +85,11 @@ void BTLDeviceSim::getHitsResponse(const std::vector<std::tuple<int, uint32_t, f
         simHitAccumulator->emplace(mtd_digitizer::MTDCellId(id, row, col), mtd_digitizer::MTDCellInfo()).first;
 
     // --- Get the simHit energy and convert it from MeV to photo-electrons
-    float Npe = 1000. * hit.energyLoss() * LightYield_ * LightCollEff_ * PDE_;
+    float Npe = convertUnitsTo(0.001_MeV, hit.energyLoss()) * LightYield_ * LightCollEff_ * PDE_;
```

Comment:
@casarsa I am not sure to understand: why don't you use directly the ```_GeV``` literal https://cmssdt.cern.ch/lxr/source/DataFormats/Math/interface/GeantUnits.h#0046 ?

Reply:
I couldn't figure out how to use _GeV with a variable besides something like hit.energyLoss()*1._GeV

Reply:
I am afraid literals just work with constants